org.virt_manager.virt-viewer.appdata.xml

```diff
@@ -0,0 +1,74 @@
+<!-- https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html -->
```

Comment:
You are not installing this file anywhere. Have you submitted it upstream too?

Reply:
Haven't yet, the last time I tried to contact upstream I got absolutely nothing back.